torch/export/graph_signature.py

```diff
@@ -85,7 +85,10 @@ class OutputSpec:
     target: Optional[str]
 
     def __post_init__(self):
-        assert isinstance(self.arg, (TensorArgument, SymIntArgument, ConstantArgument))
+        assert isinstance(
+            self.arg,
+            (TensorArgument, SymIntArgument, ConstantArgument, CustomObjArgument),
```

Comment:
What's a `CustomObjArgument`? Is this more general than torch.ScriptObject?

Reply:
This is exactly torch.ScriptObject, and we probably should re-name it that way.